### PR TITLE
fix(Parameters): add support for custom connection

### PR DESCRIPTION
**Pull request by cheikhgwane**

### CODE REVIEW

File: openhexa/sdk/pipelines/parameter.py
Comment (pvanliefland) [inline openhexa/sdk/pipelines/parameter.py:1]: @cheikhgwane I can't add my comment on that specific line but could you check lines 301-303 (def `expected_type` for the`CustomConnectionType` class? It doesn't seem right.

**Review by pvanliefland — CHANGES_REQUESTED** [review]
Hi @cheikhgwane , I think there's still an issue, see comment.

Also, we should add a test for this.

File: openhexa/sdk/pipelines/parameter.py
Comment (cheikhgwane) [reply openhexa/sdk/pipelines/parameter.py:1]: Oh yes it should return CustomConnectionType

**Review by pvanliefland — APPROVED** [review]
LGTM!

**Pull request merged**